scripts/upload-docker-readme.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# Copyright (c) 2024 The Jaeger Authors.
+# SPDX-License-Identifier: Apache-2.0
+set -euxf -o pipefail
+repo="$1"
+readme_path="$2"
+abs_readme_path=$(realpath "$readme_path")
+
+DOCKERHUB_TOKEN=${DOCKERHUB_TOKEN:-}
+dockerhub_repository="jaegertracing/$repo"
+dockerhub_url="https://hub.docker.com/v2/repositories/$dockerhub_repository/"
```

Comment:
Q: what is the story with the other registries? E.g. we also push to quay.io - does it support similar API for updating the readme?